connectors/src/resources/storage/models/connector_model.ts

```diff
@@ -105,10 +107,19 @@ ConnectorModel.init(
       type: DataTypes.DATE,
       allowNull: true,
     },
+    pausedAt: {
+      type: DataTypes.DATE,
+      allowNull: true,
+    },
   },
   {
     sequelize: sequelizeConnection,
     modelName: "connectors",
-    indexes: [{ fields: ["workspaceId", "dataSourceName"], unique: true }],
+    indexes: [
+      { fields: ["workspaceId", "dataSourceName"], unique: true },
+      {
+        fields: ["pausedAt"],
```

Comment:
Any reason to add this index?

Reply:
I was expecting to query by pausedAt = null but turns out I am not